cypress/e2e/3-testing/auth.cy.js

```diff
@@ -0,0 +1,145 @@
+//variables
+
+const loginEmailInput = '[data-cy="loginEmail"]';
+const loginPasswordInput = '[data-cy="loginPassword"]';
+const loginFormBtn = '[data-cy="loginFormButton"]';
+const loginBtn = '[data-cy="loginFormSubmit"]';
+const logOutBtn = '[data-cy="logOutButton"]';
+
+//DUMMY ACCOUNT
+const email = 'testaccount4567@stud.noroff.no';
+const password = 'Testaccount4567';
+
+
+//COMMANDS
+
+
+//Get login form command
+
+Cypress.Commands.add('getLoginForm', () => {
```

Comment:
Maybe you can move these commands into it's separate file for more structure. If you add them to the cypress/support/commands.js file it should all still work

Reply:
I agree with this comment